roles/kafka_controller/defaults/main.yml

```diff
@@ -19,7 +19,7 @@ kafka_controller_logredactor_logger_specs_list:
 
 kafka_controller_java_args:
   - "{% if kafka_controller_listeners | confluent.platform.ssl_required(kafka_controller_ssl_enabled) | bool %}-Djdk.tls.ephemeralDHKeySize=2048{% endif %}"
-  - "{% if 'GSSAPI' in kafka_controller_sasl_enabled_mechanisms or ( kraft_migration|bool and zookeeper_client_authentication_type in ['kerberos']) %}-Djava.security.auth.login.config={{kafka_controller.jaas_file}}{% endif %}"
+  - "{% if 'GSSAPI' in kafka_controller_sasl_enabled_mechanisms or ( kraft_migration|bool and zookeeper_client_authentication_type in ['kerberos', 'digest']) %}-Djava.security.auth.login.config={{kafka_controller.jaas_file}}{% endif %}"
```

Comment:
In zookeeper_client_authentication_type is mtls not supported from our side or from kafka side?

Reply:
zookeeper_client_auth mtls is supported [here](https://github.com/confluentinc/cp-ansible/blob/1072283c994fd43c8e28d5680ee1f840fcf53d0d/roles/variables/vars/main.yml#L176). However, these java args are not needed when client auth is mtls

Reply:
got it